packages/skill-lesson/path-to-purchase/pricing.tsx

```diff
@@ -140,6 +141,7 @@ export const Pricing: React.FC<React.PropsWithChildren<PricingProps>> = ({
     saleCountdownRenderer: () => null,
     teamQuantityLimit: 100,
     allowTeamPurchase: true,
+    showAllContent: true,
```

Comment:
needed a way to strip the content listing here

Total TypeScript does this already, but wasn't clear how that mechanism worked and this seemed like a sensible option that wouldn't affect other properties